src/integration-tests/introspector/introspectTest.sh

```diff
@@ -393,6 +420,48 @@ function deployCreateDomainJobPod() {
   waitForPod $pod_name
 }
 
+#
+# Create the model in image docker image
+#
+function createMII_Image() {
+  trace "Info: Create MII Image"
+
+  mkdir -p ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cp ${SCRIPTPATH}/mii/models/*  ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cd ${SCRIPTPATH}/mii/workdir || exit 1
+  export WORKDIR=${SCRIPTPATH}/mii/workdir || exit 1
+  export MODEL_IMAGE_TAG=it || exit 1
+  export MODEL_IMAGE_NAME=model-in-image || exit 1
+
+  docker rmi ${MODEL_IMAGE_NAME}:${MODEL_IMAGE_TAG} --force > /dev/null 2>&1
+
+  trace "Info: Downloading WDT and WIT..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_download.sh \
+   > ${test_home}/miibuild_download.out 2>&1 || exit 1
```

Comment:
If this fails, cat the output to stdout, and then exit after that.

Reply:
done

Reply:
I don't see any cat after the failure.  It looks like it just exits immediately without a cat of miibuild_download.out